src/functions/html.js

```diff
@@ -0,0 +1,53 @@
+// @flow
+import defineFunction, {ordargument} from "../defineFunction";
+import buildCommon from "../buildCommon";
+import ParseNode, {assertNodeType} from "../ParseNode";
+
+import * as html from "../buildHTML";
+import * as mml from "../buildMathML";
+
+defineFunction({
+    type: "html",
+    names: ["\\class", "\\cssId"],
```

Comment:
While I know mathjax calls this "cssId", I think it would be nice to give it a more generic name (i.e. "id"). My only reasoning is, if multiple renderers end up being supported, css may have no meaning (i.e. in a canvas).

Reply:
@rrandallcainc were you thinking that `id` might be useful for hit testing in canvas based renderer.

Reply:
@kevinbarabash Hit testing could be one use case. I was more thinking along the lines of being able to manipulate specific parts of the LaTeX separately. For example, let's say I had `\frac{\id{numerator}{a}}{\id{denominator}{b}}` and I wanted to switch the two. I see as id/class as general ways of targeting specific areas of the latex.